File: leetcode/P771.cs
public class Solution {
    public int NumJewelsInStones(string J, string S) {
        HashSet<char> s = new HashSet<char>();
        
        for (int i=0; i<J.Length; i++) {
            s.Add(J[i]);
        }
        
        int count = 0;
        for (int i=0; i<S.Length; i++) {
            if (s.Contains(S[i])) {
                count++;
            }
        }
        return count;
    }
}

File: leetcode/P771.java
class Solution {
  public int numJewelsInStones(String J, String S) {
    HashSet<Character> s = new HashSet<Character>();

    for (int i = 0; i < J.length(); i++) {
      s.add(J.charAt(i));
    }

    int count = 0;
    for (int i = 0; i < S.length(); i++) {
      if (s.contains(S.charAt(i))) {
        count++;
      }
    }
    return count;
  }
}